### DISCUSSION TITLE
i want to remove the handle of the ndoe

### DISCUSSION THREAD

**Discussion by tansonkk** [bcakmakoglu/vue-flow#1138] [Q&A] [answered]
is it possible to remove this connection point.

<img width="326" alt="image" src="https://github.com/bcakmakoglu/vue-flow/assets/122023942/7b9aff2b-1b20-4b39-8a40-ca838215ccaf">

**Comment by bcakmakoglu** [comment, marked as answer]
Only if you're using custom nodes - the built-in ones always have a handle which, at best, you can hide with css.
